fastlane/swift/LaneFileProtocol.swift

```diff
@@ -107,11 +111,14 @@ public class LaneFile: NSObject, LaneFileProtocol {
         let lowerCasedLaneRequested = named.lowercased()
 
         guard let laneMethod = currentLanes[lowerCasedLaneRequested] else {
-            let laneNames = self.laneFunctionNames.map { laneFuctionName in
-                if laneFuctionName.hasSuffix("lanewithoptions:") {
-                    return trimLaneWithOptionsFromName(laneName: laneFuctionName)
```

Comment:
Will removing this break any existing setups that people may have?

Reply:
I would say it will not affect the users.
I have taken the following test: (given the Fastfile.swift presented above)

<img width="832" alt="additional logs to see lane informations" src="https://user-images.githubusercontent.com/4958872/39180662-c7e530a2-47b7-11e8-9220-f428baa476fb.png">

![console result with additional logic](https://user-images.githubusercontent.com/4958872/39180676-d00f1108-47b7-11e8-9d2c-f57b29a07933.png)

Reply:
From this example, me as a user I would like to see the list of available lanes as:
```
Available lanes: first, third, fourth, hockeyApp
```

instead:

```
Available lanes: first, thirdLaneWithOpti, fourthLaneWithOpti, hockeyAppLaneWithOpti
```

Reply:
It should be consistency in the list either the lane expects parameters or not.